### PR TITLE
[Doc] Try new features before release

### PR DESCRIPTION
**Pull request by qwordy**
Add an article "Try new features before release"

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by MyronFanQiu** [comment]
@qwordy Looks .PNG cannot work. I cannot see the picture correctly in your branch.

**Comment by qwordy** [comment]
> 
> 
> @qwordy Looks .PNG cannot work. I cannot see the picture correctly in your branch.

Windows doesn't care file name case, but Github cares. Fixed.

File: doc/try_new_features_without_releasing.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Azure CLI releases new versions every 3 weeks. Many customers and developers are eager to try and test new features before the formal release. Fortunately, Azure CLI project build artifacts for every commit and PR and you can download them.
+
+## Tutorial for commit
+
+Step 1. Select a commit. Click the √ or × symbol. Click "Details" of a "build package" item.
+
+![](assets/1.PNG)
+
+Step 2. Click "View more details on Azure Pipelines".
```
Comment (MyronFanQiu) [inline doc/try_new_features_without_releasing.md]: Is it able to put a red rectangle around `View more details on Azure Pipelines`? It's quite hard to find it. same with `1 artifact`

Comment (qwordy) [reply doc/try_new_features_without_releasing.md]: Good idea!

Comment (yonzhan) [reply doc/try_new_features_without_releasing.md]: Please input the doc link to OneNote after this PR is merged.

**Comment by fengzhou-msft** [comment]
The Edge build on any branch is for internal use. The Edge build of dev branch is already documented in the main readme. Maybe a link(https://github.com/Azure/azure-cli#edge-builds) is enough. The title could be improved, maybe `Try new features before release`.

**Comment by qwordy** [comment]
> 
> 
> The Edge build on any branch is for internal use. The Edge build of dev branch is already documented in the main readme. Maybe a link(https://github.com/Azure/azure-cli#edge-builds) is enough. The title could be more clear, maybe `Try new features before release`.

Good advice.

File: doc/try_new_features_without_releasing.md (hunk 1 above)
Comment (qwordy) [reply doc/try_new_features_without_releasing.md]: > 
> 
> Is it able to put a red rectangle around `View more details on Azure Pipelines`? It's quite hard to find it. same with `1 artifact`

Red rectangle added.

Comment (qwordy) [reply doc/try_new_features_without_releasing.md]: > 
> 
> Please input the doc link to OneNote after this PR is merged.

OK.

**Pull request merged**